Makefile

```diff
@@ -0,0 +1,22 @@
+default:
+	cat Makefile
+
+synthetic_data.csv:
+	wget synthetic_data.csv https://gist.github.com/Niarfe/25100a860cbb660150a3fec87378d8ac
+
+square_wave.png:
+	gnuplot -e "set datafile separator ','; set term png; set outp
```

Comment:
oh oh, what happened here, looks like there is a new line character interrupting the command on lines 8 and 12.  🤔 I don't think that would run, although I have not tried it... can we fix these up please?

Reply:
I'm not seeing the new line character... this is what I see when I'm editing the Makefile from my bash
![image](https://github.com/stephanie-yellow/DS5111_lab_01/assets/111696716/a142563e-7957-483d-8289-aa86fa245afd)